README.md

```diff
@@ -71,22 +80,31 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+A model is more for defining the database itself, and a controller is more for defining what Rails does 
+in response to a request. It wouldn't be handled in a controller, since this would presumably apply 
+whenever a song is displayed, no matter what the request was. A model might apply, since this specifies an
+integer from 1 to 10. But ther range is not really the point of this; the point is the cnange of colors, 
+so a helper method is the best choice. 
 ```
 
 ### Question 7
 
 You clone yet another Tunr repo. Put the following commands in the correct order necessary to make the app run. Delete the one command that will not be used.
 
 ```
-$ rake db:drop
-$ rake db:create
-$ rails new . -d postgresql
-$ bundle install
+
 $ git clone git@github.com:ga-wdi-exercises/moar-tunr.git
+$ bundle install
+$ rails new . -d postgresql
+$ rake db:create
 $ rake db:migrate
-$ rails s
 $ rake db:seed
+$ rails s
+
+`rake db:drop` ia not needed, since there is no existing database to drop. It would be like the Skipper yelling, 
+"Gilligan! Drop those coconuts!" when Gilligan didn't have any coconuts. Hijinks might ensue, and Ruby on Rails 
+is known to have issues with hijinks. 
+
 ```
```

Comment:
:+1: There's actually no need for the `rails new` step since we're cloning down an existing Rails application.

In this case we do run `rake db:drop` in case we already have a database of that name with existing data. Think of it as starting fresh. It's not necessarily a required step, but one that is used in this example.